StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -266,8 +266,9 @@ extension FinancialConnectionsAPIClient: FinancialConnectionsAPI {
     }
 
     func markConsentAcquired(clientSecret: String) -> Promise<FinancialConnectionsSessionManifest> {
```

Comment:
I tried to query all the places where we return manifest, and then added the `expand` in the missing places